src/commands/logs.rs

```diff
@@ -0,0 +1,120 @@
+use anyhow::{anyhow, Result};
+
+use crate::commands::deploy::{login_connection, SPIN_DEPLOY_CHANNEL_NAME};
+use crate::commands::login::LoginConnection;
+use crate::commands::{get_app_id_cloud, get_channel_id_cloud};
```

Comment:
Can use helper for creating the client. See below for continued suggestion
```suggestion
use crate::commands::{get_app_id_cloud, get_channel_id_cloud, create_cloud_client};
```

Reply:
👍